analysis/source/simulation/designs/500_8.csv

```diff
@@ -1,2 +1,2 @@
-sample_size,500
+sample_size,1000
```

Comment:
@nateschor Is 1000 enough simulations?

Reply:
@kobleary good point. Based on the simulations [here](https://ditraglia.com/Econ103Public/Rtutorials/Rtutorial4.html#approximating-probabilities), the sample distributions are close for $N = 1,000$ and $N = 100,000$. So it looks like $X_n \xrightarrow{\text{d}} X$ where $X = U_1 + U_2$, $U_i \sim \text{Unif}\\{1,8\\}$.

I think 1,000 trials is sufficient, please let me know if you disagree?

Reply:
@nateschor sounds good. resolving this!